xhyve/xhyve.go

```diff
@@ -610,6 +613,49 @@ func (d *Driver) publicSSHKeyPath() string {
 	return d.GetSSHKeyPath() + ".pub"
 }
 
+func readLine(path string) []string {
+	lines := []string{}
+	inFile, err := os.Open(path)
+	if err != nil {
+		log.Debugf("Not able to open %s", path)
+	}
+	defer inFile.Close()
+	scanner := bufio.NewScanner(inFile)
+	scanner.Split(bufio.ScanLines)
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+	return lines
+}
+
+func extractKernelOptions(d *Driver) error {
```

Comment:
> Could you including this func to Driver method?

No, I needed it `volumeRootDir := d.ResolveStorePath(isoMountPath)` for this, otherwise I have to again need to mount and get those details. Or there any other better I can get this value?